tests/transports-governance/test_governance_integration_flow_hierarchical.py

```
@@ -0,0 +1,1498 @@
+            vk_ids.append(vk_id)
+            track_created_entity("virtual_keys", vk_id)
+
+        # Simulate peak load with concurrent operations
+        def peak_load_operation():
+            import random
+
+            operations = [
+                lambda: governance_client.get_usage_stats(),
+                lambda: governance_client.list_virtual_keys(),
+                lambda: governance_client.get_customer(customer_id),
+                lambda: governance_client.get_virtual_key(random.choice(vk_ids)),
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)